### ISSUE TITLE
How can I choose the method when I run the script ？

### ISSUE THREAD

**Issue by Shuai-Lv** [hila-chefer/Transformer-MM-Explainability#14]
CUDA_VISIBLE_DEVICES=0 PYTHONPATH=`pwd` python VisualBERT/run.py **--method=<method_name>** --is-text-pert=<true/false> --is-positive-pert=<true/false> --num-samples=10000 config=projects/visual_bert/configs/vqa2/defaults.yaml model=visual_bert dataset=vqa2 run_type=val checkpoint.resume_zoo=visual_bert.finetuned.vqa2.from_coco_train env.data_dir=/path/to/data_dir training.num_workers=0 training.batch_size=1 training.trainer=mmf_pert training.seed=1234

**Comment by hila-chefer** [comment]
Hi @Shuai-Lv, thanks for your interest!

Not sure I follow your question, you can simply exchange ‘< method_name>‘ with the name of the method you wish to use. 
I hope this helps.

**Comment by Shuai-Lv** [comment]
Thanks for you guidance, I am new to this, so the question is a little silly...
